qt6/qml/gen_qqmllist.h

```diff
@@ -29,13 +29,11 @@ typedef struct QVariant QVariant;
 #endif
 
 QQmlListReference* QQmlListReference_new();
-QQmlListReference* QQmlListReference_new2(QVariant* variant);
```

Comment:
the interesting thing about these removed overloads is that `_3` and `_4` are the same .. :shrug: